plugins/kubernetes-backend/src/service/KubernetesBuilder.ts

```diff
@@ -268,6 +311,67 @@ export class KubernetesBuilder {
     const logger = this.env.logger;
     const router = Router();
     router.use(express.json());
+    router.use(
+      createPermissionIntegrationRouter({
+        resourceType: RESOURCE_TYPE_KUBERNETES_RESOURCE,
+        // permissions?: Array<Permission>,
+        rules: [matchesJSON],
+        getResources: (resourceRefs: string[]) => {
+          return Promise.all(
+            resourceRefs.map(async (resourceRef: string) => {
+              // Parse the reference, following a format like this:
+              // "cluster:group/v1beta1.things:namespace/name"
```

Comment:
I don't know if there's some open standard for "coordinates of a k8s resource in a multi-cluster context" so I'm improvising here

Reply:
LOL reading it back, maybe a proper URL is the right thing to use here